jschon/vocabulary/__init__.py

```diff
@@ -51,6 +51,22 @@ def _bootstrap(self, value: Mapping[str, JSONCompatible]) -> None:
             for vocabulary in self.default_vocabularies:
                 self.kwclasses.update(vocabulary.kwclasses)
 
+    def get_kwclass(self, key):
```

Comment:
Please add type hints for the param and return value.

Could you also please add a short docstring for this method? I know this module is a bit light on docstrings, but ultimately anything that's part of the public API should have one.

Reply:
Will do!  I'm trying to be mindful about the type hinting but it's still new to me and sometimes I forget.  Agree about the docstrings.